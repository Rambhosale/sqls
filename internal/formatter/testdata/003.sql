SELECT
    ap.autograph_purchase_id AS "id",
    ap.order_number AS "order",
    ap.product_price AS "productPrice",
    i.name AS "influencerName",
    p.name AS "productName",
    u.email AS "email"
FROM
    autograph_purchaseASap innser
JOIN influencer AS i
    ON autograph_purchase.influencer_id = influencer.influencer_id
LEFT JOIN product AS p
    ON product.product_id = autograph_purchase.product_id
LEFT JOIN USER1 AS u
    ON USER1.user_id = autograph_purchase.user_id
