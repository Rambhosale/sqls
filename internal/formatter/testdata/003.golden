-- hoge --
SELECT
		x/*x*/,
		/*x*/y
	FROM
		zzz;
-- zzzz
SELECT
		*
	FROM
		yyy;
-- yyyy
-- hage --
SELECT
    ap.autograph_purchase_id AS "id",
    ap.order_number AS "order",
    ap.product_price AS "productPrice",
    i.name AS "influencerName",
    p.name AS "productName",
    u.email AS "email"
FROM
    autograph_purchaseASap innser
JOIN influencer AS i
    ON autograph_purchase.influencer_id = influencer.influencer_id
LEFT JOIN product AS p
    ON product.product_id = autograph_purchase.product_id
LEFT JOIN USER AS u
    ON USER.user_id = autograph_purchase.user_id
